### ISSUE TITLE
Postgres Errors Using Vagrant

### ISSUE THREAD

**Issue by LadyMozzarella** [TechEmpower/FrameworkBenchmarks#1442]
I'm getting errors while using vagrant when attempting to run any tests with Postgres after fresh vagrant up. 

`/var/log/postgresql/postgresql-9.3-main.log`:
```
2015-03-24 23:21:25 UTC LOG:  database system was shut down at 2015-03-24 23:21:25 UTC
2015-03-24 23:21:25 UTC LOG:  autovacuum launcher started
2015-03-24 23:21:25 UTC LOG:  database system is ready to accept connections
2015-03-24 23:21:25 UTC LOG:  incomplete startup packet
2015-03-24 23:21:59 UTC LOG:  received fast shutdown request
2015-03-24 23:21:59 UTC LOG:  aborting any active transactions
2015-03-24 23:21:59 UTC LOG:  autovacuum launcher shutting down
2015-03-24 23:21:59 UTC LOG:  shutting down
2015-03-24 23:21:59 UTC LOG:  database system is shut down
2015-03-24 23:22:00 GMT FATAL:  could not map anonymous shared memory: Cannot allocate memory
2015-03-24 23:22:00 GMT HINT:  This error usually means that PostgreSQL's request for a shared memory segment exceeded available memory or swap space. To reduce the request size (currently 2399051776 bytes), reduce PostgreSQL's shared memory usage, perhaps by reducing shared_buffers or max_connections.
2015-03-24 23:22:01 GMT FATAL:  could not map anonymous shared memory: Cannot allocate memory
2015-03-24 23:22:01 GMT HINT:  This error usually means that PostgreSQL's request for a shared memory segment exceeded available memory or swap space. To reduce the request size (currently 2399051776 bytes), reduce PostgreSQL's shared memory usage, perhaps by reducing shared_buffers or max_connections.
...
```

**Comment by GMLudo** [comment]
1. Could you give us RAM and swap you have in your VM ?
2. Could you increase available RAM for your vagrant VM ?

**Comment by LadyMozzarella** [comment]
[Looks like](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/deployment/vagrant-common/core.rb#L98) we're starting up with 2048MB and 0 swap.
![swapspace](https://cloud.githubusercontent.com/assets/4926965/6830111/fe54524a-d2d4-11e4-8c6a-8678f9e6f019.png)

I'm working on increasing the RAM. I'll get back to you on that one.

**Comment by LadyMozzarella** [comment]
Okay. I increased the RAM to 3022 and the postgres tests appear to be passing now. Thanks @GMLudo.

**Comment by GMLudo** [comment]
@LadyMozzarella You're welcome
FYI, sometimes it should be also a kernel setting you need to change, but normally, I've should fixed that in #1407. You can find documentation links in the comments of the Python application.
